test/runnable/cpp_abi_tests.d

```diff
@@ -94,7 +107,7 @@ template IsBoolean(T)
 
 template IsSomeChar(T)
 {
-    enum IsSomeChar = is(T==char) || is(T==dchar);
+    enum IsSomeChar = is(T==char) || is(T==wchar) || is(T==dchar) || is(T==wchar_t);
```

Comment:
use `is(T : wchar)` because an the enum `__c_wchar_t` is not the same (`==`). Ditto for `dchar`.

Reply:
Oh wow I can't read. Ignore that.